interfaces/matlab/Utility/+ct/gitCommit.m

```diff
@@ -7,5 +7,5 @@
     %     A string containing the Git commit hash for the current version of Cantera.
 
     ct.isLoaded(true);
-    v = ct.impl.getString('mCt_getGitCommit');
+    v = ct.impl.getString('mCt_gitCommit');
```

Comment:
Is there a test for this that would have caught the typo? Can we add one? I'm pretty sure we have one for Python.